scrapper.py

```diff
@@ -1,6 +1,12 @@
 """
 Scrapper implementation
 """
+import json
+import re
+import os
+import requests
+from bs4 import BeautifulSoup
+# from constants import CRAWLER_CONFIG_PATH, ASSETS_PATH
```

Comment:
we should not keep commented code. either uncomment or remove

however, those imports are rather critical

Reply:
okay